src/serializedobject.js

```diff
@@ -140,10 +140,9 @@ SerializedObject.prototype.append = function(bytes) {
   // Make sure both buffer and bytes are Array. Either could potentially be a
   // Buffer.
   if (Array.isArray(this.buffer) && Array.isArray(bytes)) {
-    // Array::concat is horribly slow where buffer length is 100 kbytes + One
-    // transaction with 1100 affected nodes took around 23 seconds to convert
-    // from json to bytes.
-    Array.prototype.push.apply(this.buffer, bytes);
+    for (var i = 0; i < bytes.length; i++) {
```

Comment:
We should put a comment here explaining why this helps. If I saw this code outside the context of this pull request, I would probably refactor it back to the way it was before.